docs/DeveloperGuide.adoc

```diff
@@ -279,51 +287,88 @@ Refer to the guide <<DevOps#, here>>.
+* has a need to keep track of Mentor assignments to Teams
+* has a need to keep track of the competition winners and prizes won
+* has a need to search for specific Mentor, Team or Participant at times
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+
+* manage contacts faster than a typical mouse/GUI driven app
+* keep track of the relationship between Participant, Team and Mentor, such that it can be referenced at times
+* store a significant number of contacts in an organised, readable manner
 
 [appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="79%",cols="15%,<20%,<35%,<37%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |Admin In-Charge |find a Entity by name |locate details of Entity without having to go through the entire list
+
+|`* * *` |Admin In-Charge |delete an Entity by name |remove entries I no longer need
+
+|`* * *` |Admin In-Charge |add an Entity by name and contact information |update the list of Entities
+
+|`* * *` |Admin In-Charge |updated an Entity by name and contact information |update the specific entries
+
+|`* * *` |Admin In-Charge |register individuals en-masse(with provided registration information) | avoid tedious manual registration
+
+|`* * *` |Admin In-Charge |keep track of winning teams and the prizes won | ensure that the prize-giving ceremony runs smoothly
+
+|`* * *` |Admin In-Charge |keep track of winning teams and the prizes won | ensure that the prize-giving ceremony runs smoothly
+
+|`* * *` |Admin In-Charge |make sure that I will be notified on any wrong commands that I type | make sure that I do not accidentally clutter up my list of entries
+
+|`* * *` |Admin In-Charge |have a readable and organised User Interface | understand the output of my commands
+
+|`* * *` |Admin In-Charge |keep track of participants who signed up late or after the event has filled up into a waitlist | manage them in case available space turns up during the Event
+
+|`* * *` |Admin In-Charge |manually match Teams to Mentor | know which Mentor is in charge of a team
+
+|`* * *` |Admin In-Charge |keep track of where each Team or Mentor is seating|usher them to their places during the actual event
+
+|`* *` |Admin In-Charge |know my sponsor's needs and arrival time | adequately cater to their needs and allocate manpower accordingly
+
+|`* *` |Admin In-Charge |keep track of inventory of swag | make sure they are adequately catered to all participants
```

Comment:
What does swag refer to here? If possible, add it into the glossary for readers to understand. Even though it is in the user guide, it has no definition.